oauth2client/service_account.py

```diff
@@ -461,3 +462,56 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+class SelfSignedJWTCredentials(object):
```

Comment:
How many users know what a JWT is? In my experience with service accounts, the previous name `SignedJWTCredentials` was confusing to users (and to contributors / Googlers, some of which accidentally created a new service account class because they didn't realize `SignedJWTCredentials` was for service accounts).